rero_ils/modules/loans/api.py

```diff
@@ -147,23 +147,30 @@ def organisation_pid(self):
         if self.get('item_pid'):
             item = Item.get_record_by_pid(self.get('item_pid'))
             return item.organisation_pid
-        return None
 
     @property
     def library_pid(self):
         """Get library PID regarding transaction location PID or location."""
         from ..items.api import Item
 
         location_pid = self.get('transaction_location_pid')
+        if location_pid:
+            return Location.get_record_by_pid(location_pid).library_pid
         item_pid = self.get('item_pid')
+        if item_pid:
+            return Item.get_record_by_pid(item_pid).holding_library_pid
 
-        if not location_pid and item_pid:
-            item = Item.get_record_by_pid(item_pid)
-            return item.holding_library_pid
-        elif location_pid:
-            loc = Location.get_record_by_pid(location_pid)
-            return loc.library_pid
-        return None
+    @property
+    def location_pid(self):
+        """Get location PID regarding transaction Location PID or location."""
```

Comment:
"""Return loan transaction_location PID of Item location."""

Reply:
done